docs/products/clickhouse/concepts/databases-and-tables.rst

```diff
@@ -10,23 +10,21 @@ Databases and tables are at the core of any **Database Management System**. Clic
 Databases
 ---------
 
-Similarly when working with any other database technology, the first step is to create the database itself. Due to security reasoning, the `admin` user is not allowed to create databases. However, you can create a new database through the web interface of `Aiven console <https://console.aiven.io/>`_:
+Creating tables in an Aiven for ClickHouse service can only be done via the Aiven platform; the `admin` user is not allowed to create databases directly for security and reliability reasons. However, you can create a new database through the web interface of `Aiven console <https://console.aiven.io/>`_:
```

Comment:
This is a bit confusing - "Creating tables" - we're talking about databases here
so it probably should be "Creating databases in an Aiven for ClickHouse service can only be done via "
To make it more specific I'd replace platform with console.

Reply:
Good catch!